.github/workflows/cicd-main.yml

```
@@ -0,0 +1,98 @@
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  deploy:
```

actionlint (1.7.11)

[error] "steps" section is missing in job "deploy"

(syntax-check)